app/assets/javascripts/templates/schedule.hbs

```diff
@@ -4,6 +4,7 @@
       <span id="leftSwitch" class="scheduleswitch">&#9664;</span>
       <span id="scheduleNum">0</span> / {{schedules.length}}
       <span id="rightSwitch" class="scheduleswitch">&#9654;</span>
+      <span id="clearSwitch" class="scheduleswitch">Clear All Selections</span>
```

Comment:
the `scheduleswitch` class / switch naming convention refers to the left and right buttons, please rename appropriately.